src/task.test.js

```diff
@@ -8,6 +9,13 @@ const dom = new JSDOM('<!DOCTYPE html><html><head></head><body></body></html>');
 global.window = dom.window;
 global.document = dom.window.document;
 
+beforeAll(() => {
+  Object.defineProperty(window, 'location', {
+    value: { reload: jest.fn() },
+    writable: true,
+  });
+});
+
```

Comment:
- [ ] For the testing `addTask` function you did great by expecting the `tasks` array in the localstorage to grow in length by one, however from the project requirement you suppose to test the dom-manipulation as well 👍🏼 this is noted from the screenshot below 💯 

![Screenshot from 2023-04-06 22-08-32](https://user-images.githubusercontent.com/68030297/230494412-a4f2487e-d9ef-4402-8822-928b132136fe.png)

### Suggestions 
![Screenshot from 2023-04-06 22-13-00](https://user-images.githubusercontent.com/68030297/230495123-5c832472-1364-485a-b06d-021cfb0452c8.png)

- Here is your code after adding a new task you can consider testing the DOM by following:
```javascript
addTask(newTask);
renderTasks(); // Render the task into the dom
const newTaskDom = document.querySelector('li'); // select the first and only task from the dom
...

expect(newTaskDom).toBeTruthy();
```